README.md

```diff
@@ -87,6 +87,10 @@ ___
 The UI is a React-based Web App available in the `SearchUI` folder. Navigate to [**SearchUI/README.md**](SearchUI/README.md) for more details on the web app. 
 
 
+## Discliamer
+
+This repo has been tested with document sets as big as 2,000 documents. If you have a larger dataset or run into any issues with this repo, please open an [issue](https://github.com/Azure-Samples/search-qna-maker-accelerator/issues) and we will get back you.
```

Comment:
You are missing a preposition: "with or "to".  "..and we will get back **with** you."

Reply:
Done, thank you!